README.md

```diff
@@ -1,18 +1,18 @@
-# C# Code Analyzer
+# Infer# GitHub Action
 
-**C# Code Analyzer** is an interprocedural and scalable static code analyzer for C#. Via the capabilities of Facebook's [Infer](https://fbinfer.com/), this tool detects null pointer dereferences and [resource leak](Examples/ResourceLeak/README.md). Its source code can be found [here](https://github.com/microsoft/infersharp).
+**Infer#** is an interprocedural and scalable static code analyzer for C#. Via the capabilities of Facebook's [Infer](https://fbinfer.com/), this tool detects null pointer dereferences and [resource leak](Examples/ResourceLeak/README.md). Its source code can be found [here](https://github.com/microsoft/infersharp).
 
-![Sample Report](https://github.com/microsoft/CSharpCodeAnalyzer/blob/main/assets/samplereport.png)
+![Sample Report](https://github.com/microsoft/infersharpaction/blob/main/assets/samplereport.png)
```

Comment:
Do we want to replace this screenshot? It has "C# Code Analyzer" brand name on multiple places.

Reply:
Good catch!